File: app/src/main/java/net/todd/mavelheroes/CharacterThumbnail.java
package net.todd.mavelheroes;

import com.google.gson.annotations.SerializedName;

public class CharacterThumbnail {
    @SerializedName("path")
    String path;
    @SerializedName("extension")
    String extension;

    public CharacterThumbnail(String path, String extension) {
        this.path = path;
        this.extension = extension;
    }

    public String getImagePath() {
        return path + "." + extension;
    }
}

File: app/src/main/java/net/todd/mavelheroes/net/todd/mavelheroes/data/CharacterThumbnail.kt
package net.todd.mavelheroes.net.todd.mavelheroes.data

import com.google.gson.annotations.SerializedName

class CharacterThumbnail(@SerializedName("path")
                         internal var path: String, @SerializedName("extension")
                         internal var extension: String) {

    val imagePath: String
        get() = path + "." + extension
}
